dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageContainerForm.java

```diff
@@ -0,0 +1,94 @@
+package com.dotcms.rest.api.v1.page;
+
+import com.dotcms.repackage.com.fasterxml.jackson.core.JsonParser;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.DeserializationContext;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.JsonDeserializer;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.JsonNode;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+/**
+ * {@link PageResource#addContent(HttpServletRequest, HttpServletResponse, String)}'s form
+ */
+@JsonDeserialize(using = PageContainerForm.ContainerDeserialize.class)
+public class PageContainerForm {
+
+    private final List<ContainerEntry> entries;
+    private final String requestJson;
+
+    public PageContainerForm(final List<ContainerEntry> entries, final String requestJson) {
+        this.entries = entries;
+        this.requestJson = requestJson;
+    }
+
+    public List<ContainerEntry> getContainerEntries() {
+        return entries;
+    }
+
+    public String getRequestJson() {
+        return requestJson;
+    }
+
+    static final class ContainerDeserialize extends JsonDeserializer<PageContainerForm> {
+
+        private static final String CONTAINER_ID_ATTRIBUTE_NAME = "id";
+        private static final String CONTAINER_UUID_ATTRIBUTE_NAME = "uuid";
+        private static final String CONTAINER_CONTENTLETSID_ATTRIBUTE_NAME = "contentletsId";
+
+        @Override
+        public PageContainerForm deserialize(final JsonParser jsonParser,
+                                             final DeserializationContext deserializationContext)
+                throws IOException {
+
+            final JsonNode jsonNode = jsonParser.readValueAsTree();
+            final List<ContainerEntry> entries = new ArrayList<>();
+
+            for (JsonNode jsonElement : jsonNode) {
+                final String containerId = jsonElement.get(CONTAINER_ID_ATTRIBUTE_NAME).asText();
+                final String containerUUID = jsonElement.get(CONTAINER_UUID_ATTRIBUTE_NAME).asText();
+                final ContainerEntry containerEntry = new ContainerEntry(containerId, containerUUID);
+
+                final JsonNode containerNode = jsonElement.get(CONTAINER_CONTENTLETSID_ATTRIBUTE_NAME);
+
+                containerNode.forEach(contentId -> containerEntry.addContentId(contentId.textValue()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'contentId' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)